source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -226,18 +231,58 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol, SelectActionHa
         layer?.backgroundColor = ColorUtils.hoverColorOnMouseEnter().cgColor
     }
     
-    func setErrorMessage(with title: String, for view: InputHandlingViewProtocol) {
-        guard errorMessageView == nil else { return }
+    private func staticTextField() -> NSTextField {
         let textField = NSTextField()
-        textField.stringValue = title
-        textField.isHidden = true
+        textField.allowsEditingTextAttributes = true
         textField.isEditable = false
         textField.isBordered = false
         textField.isSelectable = true
         textField.backgroundColor = .clear
+        return textField
+    }
+    
+    func configureInputElements(element: ACSBaseInputElement, view: NSView) {
+        setupLabel(for: element)
+        addArrangedSubview(view)
+        if view is ACRContentStackView {
+            view.widthAnchor.constraint(equalTo: widthAnchor).isActive = true
+        }
+        setupErrorMessage(element: element, view: view)
+    }
+    
+    private func setupLabel(for element: ACSBaseInputElement) {
+        guard renderConfig.supportsSchemeV1_3, inputLabelField?.stringValue != element.getLabel() else { return }
+        if let label = element.getLabel(), !label.isEmpty {
```

Comment:
`let label = element.getLabel(), !label.isEmpty` these can be clubed in the guard as well?